### ISSUE TITLE
docs: 📚 evaluate WCAG required defaults to be shown in storybook

### ISSUE THREAD

**Issue by karlbaumhauer** [solid-design-system/solid#611]
## User Story
In case we have a required property in a component which can be filled by an attribute OR a slot (eg. label on `sd-input`), we need to make sure our users know about the WCAG requirements.

Therefore we need to add a "WCAG Requirements" section into the description of each component and highlight the required fields. In addition to this, we need to refactor our stories to only show samples with the required properties.

- [ ] sd-select
- [ ] sd-radio
- [ ] sd-radio-group
- [ ] sd-checkbox
- [ ] sd-checkbox-group
- [ ] sd-switch
- [ ] sd-textarea
- [ ] sd-input
- [ ] sd-range
- [ ] ... Anything missing?

## DoR
- [x] Item has business value
- [x] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant E2E tests (Features, A11y, Bug fixes) are created/updated
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

**Comment by coraliefeil** [comment]
I added all form elements that need this section to the description.
So s.o. from dev can pick up this task.

**Comment by karlbaumhauer** [comment]
@coraliefeil just to clarify. Is this already done in Figma or did you just collect the components?

**Comment by coraliefeil** [comment]
Listet the comps. Thought the task to go through all of them should be conducted in figma and storybook in the next iteration. Or am I mistaken? Feel free to reassign it to me in case.

**Comment by karlbaumhauer** [comment]
perfect. Thats what I thought as well but just wanted to clarify.

Ticket is ready and assigned to iteration 9.

**Comment by mariohamann** [comment]
Will be done in big A11y docs refactor.
